frontend/package.json

```diff
@@ -24,7 +24,7 @@
     "build": "react-scripts build -- ",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint src/ --ext .jsx,.js,.ts,.tsx ",
+    "lint": "eslint src/ --config .eslintrc.prod.js --ext .jsx,.js,.ts,.tsx ",
```

Comment:
So the prod eslint rules will be used during a github action. When will the softened rules be used?

Reply:
The softened rules are used in dev.